src/utils/searchMulti.js

```diff
@@ -0,0 +1,10 @@
+import { key } from "../secrets";
+const api = `https://api.themoviedb.org/3/search/multi?api_key=${key}&language=en-US&&page=1&include_adult=true`;
+
+const searchMulti = (setMulti) => {
```

Comment:
I don't think this file is being used at all. Delete it?

Reply:
deleted